src/store/actions.js

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
Maybe instead of dividing store into 4 files like this we could have one index.js file with multiple store modules for separated views and then one index.js in the root with all project store modules? What do you think?

Reply:
Ok, I will do that in next feature. For now I think we could stay in this base config.

Reply:
I didn't say you have to do it. But rather what do you think to swithing to this kind of setup? :)

Reply:
But i will do it :D But for now it is base setup because we don't have any feature with separated view. (not included Auth)